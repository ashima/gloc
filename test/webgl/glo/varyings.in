varying vec3 gP;

vec3 get_gP() { return gP; }
void set_gP(vec3 p) { gP = p; }
