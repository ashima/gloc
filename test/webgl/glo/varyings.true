{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": { "copyright": [ 2012, [ "", "" ] ] },
  "units": [
    {
      "outsym": [ "set_gP", "get_gP", "gP" ],
      "source":
        "#line 1 GLOC_0\nvarying vec3 gP;\n\nvec3 get_gP() { return gP; }\nvoid set_gP(vec3 p) { gP = p; }"
    }
  ],
  "linkmap": { "0": "[stdin]#n=0" }
}
