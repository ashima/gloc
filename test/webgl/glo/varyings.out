{
  "glo": [ 1, 0, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": { "copyright": [ 2012, [ "", "" ] ] },
  "units": [
    {
      "vary": [ "gP" ],
      "outsym": [ "set_gP", "get_gP", "gP" ],
      "source":
        "#line 1 GLOC_0\nvarying vec3 gP;\n\nvec3 get_gP() { return gP; }\nvoid set_gP(vec3 p) { gP = p; }"
    }
  ],
  "linkmap": { "0": "#n=0" }
}
